demo-private.html

```diff
@@ -4,9 +4,10 @@
 <head>
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
   <title>google-sheets Demo</title>
-  <script src="../webcomponentsjs/webcomponents.min.js"></script>
+  <script src="../webcomponentsjs/webcomponents.js"></script>
```

Comment:
Alright. Just pointing out that the 'recommended' way of using git to work on polymer doesn't work with the `-min`.